packages/components/src/Auth.purs

```diff
@@ -43,3 +48,20 @@ requireToken =
   loadToken >>= case _ of
     Nothing -> liftEffect $ throw "Did not find uuid/token in local storage."
     Just loginResponse -> pure loginResponse
+
+setAuthCookies :: Aff Unit
```

Comment:
As this is only for mosaico, we could call it `setMosaicoAuthCookies` to prevent future confusion 🙂

Reply:
Function name disambiguated.